src/main/kotlin/assignment/Assignment.kt

```diff
@@ -21,13 +24,35 @@ fun main() {
 }
 
 fun readResponses(): List<Response> {
-    TODO()
+    var returnList = mutableListOf<Response>()
+    val sc = Scanner(File("responses.txt"))
```

Comment:
The file path is complete. This won't read the file present inside the `assignment` directory.